src/ORM/Query.php

```diff
@@ -1275,4 +1275,22 @@ protected function _decorateResults($result)
 
         return $result;
     }
+
+    /**
+     * Selects all fields in the schema with the exception of the fields you pass
+     * in the $excludedFields parameter.
+     *
+     * @param array $excludedFields fields you don't want to select
+     * @param bool $overwrite whether to reset fields with passed list or not
+     * @return $this
+     */
+    public function selectAllExcept(array $excludedFields = [], $overwrite = false)
+    {
+        $allFields = $this->repository()->getSchema()->columns();
+        $aliased = $this->aliasFields($allFields, $this->repository()->alias());
```

Comment:
I need the aliased fields otherwise it won't work if I pass aliased fields to as $excludedFields